pages/add-sponsor.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { AddSponsor } from '../client/components';
+
+const AddSponsorPage = () => {
+  // TODO - Dynamically get Chapter and EventId s
+  const eventId = '123abc';
+  const chapterId = 'abc123';
+
+  return (
+    <div>
```

Comment:
instead of wrapping component over `<div>` we can use `<> </>`. It's more of a react way of doing it

Reply:
Yup I originally did this and my editor threw me a bunch of errors about something so I assumed it wasn't supported in TS... but now I tried again and it just... works. 
Sometimes I hate code.